### DISCUSSION TITLE
Clarify the usage of the symbol "@" in the keys on the guide async-storage/website/docs/Usage.md

### DISCUSSION THREAD

**Discussion by KirillBorunov** [react-native-async-storage/async-storage#956] [Q&A] [answered]
### What happened?

In the examples you are using the symbol "@" in the key argument, but there is no explain why, and if this is required.

https://github.com/react-native-async-storage/async-storage/blob/ba983157c127f77b5bb0ece3c09e3077c978bc6d/website/docs/Usage.md?plain=1#L26

https://github.com/react-native-async-storage/async-storage/blob/ba983157c127f77b5bb0ece3c09e3077c978bc6d/website/docs/Usage.md?plain=1#L38

https://github.com/react-native-async-storage/async-storage/blob/ba983157c127f77b5bb0ece3c09e3077c978bc6d/website/docs/Usage.md?plain=1#L54

https://github.com/react-native-async-storage/async-storage/blob/ba983157c127f77b5bb0ece3c09e3077c978bc6d/website/docs/Usage.md?plain=1#L70

### Version

https://react-native-async-storage.github.io/async-storage/docs/usage

### What platforms are you seeing this issue on?

- [X] Android
- [X] iOS
- [X] macOS
- [X] Windows
- [X] web

### System Information

```true
Issue is about the docs.
```

### Steps to Reproduce

Not needed.

**Comment by krizzu** [comment]
It's nothing special about it, it's fine to ignore

**Reply by KirillBorunov** [reply]
Then I suggest to remove it from the examples or clarify, because it feels like a strange required thing (at least for a novice like me).
My first thought was it is required on some of the platforms, but searched all the docs about it and found no information about it. Then I created this issue.

**Reply by fayaadbsa** [reply]
Yes, I agree with it, sometimes it make me confuse about what's the best practices of naming the key in AsyncStorage.

**Reply by tido64** [reply, marked as answer]
Hi folks, we've just changed it to `my-key` instead. Hope that's clearer.

**Comment by krizzu** [comment]
With recent changes to docs (thanks @tido64) I hope it'll be more clear now 🙏

**Reply by KirillBorunov** [reply]
Yes! Thank you!. Beginners will not be confused now 😉
